<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAman</class>
 <widget class="QDialog" name="DialogAman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ATCAmanDisplay" name="amanDisplay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>700</width>
     <height>795</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView
{
	background-color: #000000;
	border-style: outset;
	border-width: 2px;
	border-color: #3e3e3e;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClock">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>825</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 19pt;
	color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string>--:--:--</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>825</y>
     <width>290</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-style: outset;
	border-width: 2px;
	border-color: #3e3e3e;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStats">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>290</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameRight">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>825</y>
     <width>290</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-style: outset;
	border-width: 2px;
	border-color: #3e3e3e;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonClearAll">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>0</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>CLR
ALL</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>115</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTimeEdit:enabled
{
	color: #c8c8c8;
	background-color: #000000;
	border-style: outset;
	border-width: 2px;
	border-color: #3e3e3e;
	selection-background-color: #060064;
	font: 18px;
}

QTimeEdit:disabled
{
	color: #323232;
	background-color: #000000;
	border-style: outset;
	border-width: 2px;
	border-color: #3e3e3e;
	font: 18px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>0</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>CLR</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>35</y>
      <width>205</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #191919;
    height: 8px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #282828, stop:1 #323232);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}

QSlider::groove:horizontal:disabled{
    border: 1px solid #191919;
    height: 8px;
    background: #191919;
    margin: 2px 0;
}

QSlider::handle:horizontal:disabled {
    background: #323232;
    border: 1px solid #191919;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSooner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>57</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel:enabled
{
	border: none;
}

QLabel:disabled
{
	color: #323232;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Sooner</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLater">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>57</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel:enabled
{
	border: none;
}

QLabel:disabled
{
	color: #323232;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Later</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetRTA">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>0</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonMeteringFix">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>875</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt;
color: #c8c8c8;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCAmanDisplay</class>
   <extends>QGraphicsView</extends>
   <header>atcamandisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
